src/eprunner.py

```diff
@@ -49,6 +50,14 @@ def run_simulation(self):
             precess run object, which contains all stdout/stderr, will be saved by the save_log method
 
         """
+
+        # copy weather file to the simulation output folder to allow simulations in parallel
+        Path(self.output_path).mkdir(parents=True, exist_ok=True)
+        shutil.copyfile(
+            self.weather_path, self.output_path + "/" + self.weather_path.split("/")[-1]
+        )  # weather_path should perhaps be renamed
+        self.weather_path = self.output_path + "/" + self.weather_path.split("/")[-1]
+
```

Comment:
Copy the weather file over to the "injected" folder to allow running simulation using multiple threads.